### ISSUE TITLE
Push Registration cannot be liaded

### ISSUE THREAD

**Issue by hemee** [ChatSecure/ChatSecure-iOS#803]
I found some strange bug: if you go to server information, you'll see that Push Registration loads too slow.
![](http://i.imgur.com/Cx5sxnP.jpg)
After that, the yellow triangle occurs.
![](http://i.imgur.com/pS1Xhp4.jpg)
I don't remind this bug in the previous build (93).

**Comment by chrisballinger** [comment]
You're right, I can confirm this, but turns out it's not from the latest update.

The problem is that https://push.chatsecure.org/api/v1/ seems to be going down periodically. It appears that we went over our free tier limit for keen.io server monitoring, and they "helpfully" throw errors and break our application when that happens.

**Comment by vitalyster** [comment]
Same issue, also sometimes it shows "incompatible server" and "used tokens" are constantly grow. Also notifications sometimes arrive in 3-5 minutes after message, while notifications for other apps are instant

**Comment by chrisballinger** [comment]
The used tokens count should constantly grow, that's part of the design and is harmless.

The delayed pushes are caused by using low priority background pushes. We don't have VoIP so we can't use high priority background pushes.

**Comment by vitalyster** [comment]
I guess they delayed just because appserver crashed, because I have no delays with test app without any voip configuration
